tensorflow/BUILD

```diff
@@ -730,6 +738,7 @@ tf_cc_shared_object(
         "//conditions:default": [
             "-z defs",
             "-s",
+            "-Wl,-soname,libtensorflow_cc.so",
```

Comment:
Looks like this line should be on the build target below (for the C++ API), and this one should be libtensorflow.so.

Would it make sense to instead modify the [tf_cc_shared_object rule definition](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tensorflow.bzl#L285) to set this automatically based on the last bit of the "name=" field, like we do for the install_name on OSX? This would also catch a few contrib shared objects.

Reply:
Thanks, good point, done